fastlane/Fastfile

```diff
@@ -89,17 +95,20 @@ platform :ios do
   desc "Release pods"
   private_lane :release_pods do
     AmplifyPods.pods.each do |pod|
-      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
+      if pod[:no_push]
+        next
+      end
 
-      UI.message("Pushing pod #{pod[:spec]}")
+      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
 
-      dupe_error = "[!] Unable to accept duplicate entry"
+      UI.message "Pushing pod #{pod[:spec]}"
 
-      begin
-        sh('bundle', 'exec', 'pod', 'trunk', 'push', pod_path, '--allow-warnings', '--synchronous')
-      rescue => ex
-        unless ex.to_s.include? dupe_error
-          raise Exception.new "Unrecoverable error in pod push trunk"
+      with_retry do
+        begin
+          sh('bundle', 'exec', 'pod', 'trunk', 'push', pod_path, '--allow-warnings', '--synchronous')
+        rescue Exception => exception
+          raise unless exception.message =~ /Unable to accept duplicate entry/i
+          UI.warn("Version already exists. Ignoring")
```

Comment:
nit: include the exception in the message to help potential troubleshooting efforts